app/assets/stylesheets/common/_constants.scss

```diff
@@ -3,6 +3,7 @@
           $radius: 5px;
       $time-quick: 0.25s;
      $time-medium: 0.35s;
+            $black: #000000;

```

Comment:
Color `#000000` should be written as `#000`<br>Line should be indented 0 spaces, but was indented 12 spaces